launch/essentials/rgbd_to_ros.launch

```diff
@@ -21,7 +21,7 @@
     </group>
 
 	<!-- People recognition -->
-	<node name="people_detector" pkg="people_detector" type="people_detector" output="log" machine="$(arg machine)" respawn="true">
+	<node name="people_recognition_3d_node" pkg="people_recognition_3d" type="people_recognition_3d_node" output="log" machine="$(arg machine)" respawn="true">
```

Comment:
Why exactly is people_recognition in `rgbd_to_ros.launch`? It is already in `image_recognition.launch`

Reply:
This stuff was used on AMIGO&SERGIO. I don't want to break everything.

Reply:
This is already broken as people_recognition_3d_node depends on 5 other nodes before it can work. So there is no point in having it here.

Reply:
Instead of launching just one node here, it will be better to include the people_recognition launch file you just created.

Reply:
Why?? You want to seperate it. As it is completely independent.

This node is not lonely by being all by himself.